src/pages/news/news-page/news.js

```diff
@@ -1,15 +1,15 @@
 import React, { useEffect } from 'react';
 import { connect } from 'react-redux';
-import { watchNewsLoad } from '../../../redux/news/news.actions';
+import { getNews } from '../../../redux/news/news.actions';
 import { useStyles } from './news.style';
-import NewsPageItem from '../news-list';
+import NewsPageItem from '../news-item';
 import { LANGUAGE } from '../../../configs';
 
-const NewsPage = ({ watchNewsLoad, list }) => {
+const NewsPage = ({ getNews, list }) => {
   useEffect(() => {
-    watchNewsLoad();
+    getNews();
     window.scrollTo(0, 0);
-  }, [watchNewsLoad]);
+  }, [getNews]);
```

Comment:
Why do we pass getNews that is a function? Is it going to change?

Reply:
done? deleted dependency array